index.html

```diff
@@ -5,9 +5,59 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600;700;800&family=Mulish:wght@300;400;500;600;700&display=swap" rel="stylesheet">
     <title>Screeno</title>
 </head>
 <body>
-
+    <section class="main-section">
```

Comment:
Jeśli main-section to ta pierwsza górna sekcja to podpowiem Ci że nazywamy ją "Hero section" więc (to nie błąd) jak chcesz to możesz nazwę dopasować i nazwać hero a nie main